src/lib/migration/main_workflow.rb

```diff
@@ -25,6 +25,9 @@
 Yast.import "Sequencer"
 Yast.import "Update"
 Yast.import "Report"
+Yast.import "Popup"
```

Comment:
Ah, good point! I moved the `Popup` call but forgot to move the `import` as well, thanks for finding this.